lib/msprime.c

```diff
@@ -326,11 +326,19 @@ msp_set_gene_conversion_map(msp_t *self, size_t size, double *position, double *
     return ret;
 }
 
+int
+msp_set_gene_conversion_constant(msp_t *self, bool is_constant)
```

Comment:
I don't think there's much point in adding this infrastructure in - we're just going to delete it as soon as we have a good solution anyway.